.ruby-version

```diff
@@ -1 +1 @@
-2.1.0
+2.1
```

Comment:
I'm going to change this to `2.1.6`, to match the ruby version we use elsewhere.

In this case (Slices) setting this isn't as critical as other projects - it's a library, so we support multiple versions (see #108), so as long as we use one of those it should be ok. So it makes sense to just use a version we probably have installed already!